Sources/SwiftSyntaxBuilder/generated/ResultBuilders.swift

```diff
@@ -188,93 +41,21 @@ public extension ArrayElementListSyntax {
 }
 
 @resultBuilder
-public struct AttributeListBuilder {
-  /// The type of individual statement expressions in the transformed function,
-  /// which defaults to Component if buildExpression() is not provided.
-  public typealias Expression = AttributeListSyntax.Element
-  
-  /// The type of a partial result, which will be carried through all of the
-  /// build methods.
-  public typealias Component = [Expression]
-  
+public struct AttributeListBuilder: ListBuilder {
   /// The type of the final returned result, which defaults to Component if
   /// buildFinalResult() is not provided.
   public typealias FinalResult = AttributeListSyntax
   
-  /// Required by every result builder to build combined results from
-  /// statement blocks.
-  public static func buildBlock(_ components: Self.Component...) -> Self.Component {
-    return components.flatMap {
-      $0
-    }
-  }
-  
   /// If declared, provides contextual type information for statement
   /// expressions to translate them into partial results.
```

Comment:
Similarly here, I don’t think the comments provide much value and I would just remove them.